cranelift/codegen/Cargo.toml

```diff
@@ -72,7 +73,7 @@ all-arch = [
 ]
 
 # For dependent crates that want to serialize some parts of cranelift
-enable-serde = ["serde"]
+enable-serde = ["serde", "bincode", "regalloc/enable-serde"]
```

Comment:
I think the `bincode` dependency should be behind a separate feature flag. You may want to use `enable-serde` for clif ir serialization without using snapshotting.